package.json

```diff
@@ -25,7 +25,7 @@
         "jest": "29.7.0",
         "jest-sonar": "0.2.16",
         "prettier": "3.3.3",
-        "pretty-quick": "3.3.1",
+        "pretty-quick": "4.0.0",
```

Comment:
Are package updates necessary for the fix? If not I would say we could have a separate PR for that.

Reply:
Yep, approved by @devinea